src/elfpy/simulators.py

```diff
@@ -40,7 +40,7 @@ def __init__(
         self.config.check_vault_apr()
         # NOTE: lint error false positives: This message may report object members that are created dynamically,
         # but exist at the time they are accessed.
-        self.config.freeze()  # pylint: disable=no-member # type: ignore
+        self.config.freeze()  # type: ignore
```

Comment:
we no longer need the pylint disabled?  if so that's cool

Reply:
it seems to be the case, but only in a few cases, which I can't quite figure out why lol. like in the main package, it seems to be not needed. but in the test classes, it is.